tests/testsuite/namespace.go

```diff
@@ -249,6 +251,28 @@ func CleanNamespaces() {
 	}
 }
 
+// waitForNamespaceSCCAnnotations waits up to 30s for the cluster-policy-controller to add the SCC related
+// annotations to the provided namespace.
+func waitForNamespaceSCCAnnotations(c kubecli.KubevirtClient, nsName string) {
```

Comment:
Can we generalize it and passing a mapping with the annotation you want to check?

Reply:
We can, but I refrained to do so because:
1. in this case we only wait for the specific key, and don't care for the value, which is not a very general scenario..
2. I don't think waiting for annotations is something common that it would deserve a generalization. Do you know of other cases where this behavior is needed?